src/main/java/org/folio/marccat/dao/persistence/AuthorityHeadingTag.java

```diff
@@ -89,10 +89,6 @@ public boolean isAbleToBeDeleted() {
     return false;
```

Comment:
Remove this unused import 'java.util.Set' and Remove this unused import 'java.util.TreeSet'.

Reply:
I have removed it